docker-compose.yml

```
@@ -0,0 +1,14 @@
+version: "3"
+services: 
+  gosh:
+    container_name: gosh
+    stdin_open: true
+    build:
+      context: .
+      dockerfile: Dockerfile
+    tty: true
+    volumes: 
+      - type: bind
+        source: "."
+        target: "/root/app"
+    working_dir: /root/app
```

もしDockerfile修正したらここも修正お願いします🙏
あとダブルクオーテーションはない方に統一しましょう！
```suggestion
        source: .
        target: /opt/gosh
    working_dir: /opt/gosh
```

---

[こちら](https://github.com/product-dev-group/gosh/pull/1#discussion_r1103718571)で修正しました！